lib/tpl/dokuwiki/css/_footnotes.css

```diff
@@ -5,8 +5,15 @@
 /*____________ footnotes inside the text ____________*/
 
 /* link to footnote inside the text */
-.dokuwiki sup a.fn_top {
+.dokuwiki a.fn_top {
+    font-weight: bold;
+	vertical-align: super;
+	font-size: 66.7%;
+}
+.dokuwiki a.fn_top::after {
+	content: ')';
```

Comment:
If we agree that a closing parenthesis is bad style and should not be used, I'm happy to drop it.

Reply:
I would still have the parenthesis - at least in the default template - because it also makes it easier to click on the footnote, or hover to open the popup. However, by moving the bracket to CSS instead of hard-coding it, it becomes possible to make other design choices in via custom CSS.